### PR TITLE
adding june newsletter

### PR DESCRIPTION
**Pull request by nbush**
& testing netlify

### CODE REVIEW

File: help/images/newsletters/2023-06/select-row-highlighting.gif
Comment (anaisconce) [inline help/images/newsletters/2023-06/select-row-highlighting.gif:1]: Love this gif! The gif ends with clicking into rows 13 and 14 which are blank. Could the gif be trimmed to exclude that bit of blank space at the end?

File: help/newsletters/2023-06.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+### Highlighting for selector rows
+
+A small but mighty fix. Grist now highlights the selected row linked to widgets on a page.
+
+![Selector row highlighting](../images/newsletters/2023-06/select-row-highlighting.gif)
+
+## Community Highlights
+
+* @wunter8 has created a [calendar widget](https://community.getgrist.com/t/simple-calendar-widget/2676) that is a great example of what [custom widgets](https://support.getgrist.com/widget-custom/) can do. Who knows, maybe this could inspire an official calendar widget? 👀 (@ToJans also shared a calendar widget [back in March](https://support.getgrist.com/newsletters/2023-03/#custom-widget-calendar-view).)
```
Comment (anaisconce) [inline help/newsletters/2023-06.md:111]: Might be better if these links open in a new tab? Append `{:target="\_blank"}` to the url.

Same comment for the links in the bullet points and paragraph below.

The one exception might be the links that stay in the support site (to custom widgets and March newsletter).

File: help/images/newsletters/2023-06/select-row-highlighting.gif
Comment (nbush) [reply help/images/newsletters/2023-06/select-row-highlighting.gif:1]: done!

File: help/newsletters/2023-06.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+In July we’ll rebuild our [Digital Sales CRM template](https://templates.getgrist.com/eVgQezBkmQVc/Digital-Sales-CRM){:target="\_blank"}. This template contains common structural patterns used in Grist documents for a variety of use cases. It also has common, simple examples of lookup formulas, and analyzes the same sales data two ways: lifetime revenue by customer and lifetime revenue by product.
+
+**Thursday July 20th at 3:00pm US Eastern Time.**
+
+[SIGN UP FOR JULY'S WEBINAR](https://www.getgrist.com/webinars/webinar-common-structures-in-grist/?utm_source=support-site&utm_medium=newsletter&utm_campaign=build-webinar&utm_term=july-2023&utm_content=){:target="\_blank"}
+{: .grist-button}
+
+### Deconstructing the Software Deals Tracker Template
+
+In June we rebuilt the [Software Deals Tracker](https://templates.getgrist.com/viyGsuqvNF1D/Software-Deals-Tracker). This template has multiple pages and widgets focused on different workflows, and yet the document is built on only one data table. We show you why to help you anticipate your own use cases where all you need is one table with multiple views.
```
Comment (anaisconce) [inline help/newsletters/2023-06.md:130]: Same comment re: Software Deals Tracker opening in another window.

File: help/newsletters/2023-06.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+[![Field Trip Planner template](../images/newsletters/2023-06/field-trip-planner.png)](https://templates.getgrist.com/uqUVorrMc23r/Field-Trip-Planner/){:target="\_blank"}
+
+[GO TO TEMPLATE](https://templates.getgrist.com/uqUVorrMc23r/Field-Trip-Planner/){:target="\_blank"}
+{: .grist-button .grist-button-tight}
+
+### Nutrition Tracker
+
+Track your nutrition, macros, and calories with this simple template created by user Jawaad Mahmood. 🍏
+
+Check out [Jawaad's showcase post in our community](https://community.getgrist.com/t/create-a-nutrition-tracker-for-tracking-calories-and-macro-nutrients/2483).
```
Comment (anaisconce) [inline help/newsletters/2023-06.md:150]: Same comment re: showcase link.

**Review by anaisconce — COMMENTED** [review]
Looks good! Some minor tweaks.

File: help/newsletters/2023-06.md (hunk 1 above)
Comment (nbush) [reply help/newsletters/2023-06.md:111]: Will change. I think it makes sense if all links open in new tabs.

**Pull request merged**